File: codes_python/A1-018.py
roman = {i: r for i, r in enumerate(
    ["","I","II","III","IV","V","VI","VII","VIII","IX"])}
n = int(input())
if n <= 0:
    print("Error : Please input positive number")
elif n <= 9:
    print(roman[n])
else:
    print("Error : Out of range")

File: codes_c/A1-018.c
#include <stdio.h>
int main()
{
    int n;
    scanf("%d", &n);
    if (n < 0)
        printf("Error : Please input positive number\n");
    else if (n == 0 || n > 9)
        printf("Error : Out of range\n");
    else
    {
        char *r[] = {"I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IX"};
        printf("%s\n", r[n - 1]);
    }
    return 0;
}
